informal_update/models.py

```diff
@@ -0,0 +1,151 @@
+from api.storage import get_storage
+
+
+# Create your models here.
+class ReferenceUrls(models.Model):
+    url = models.URLField()
+
+
+class InformalReferences(models.Model):
+    date = models.DateTimeField(verbose_name=_('date'), blank=True)
+    source_description = models.CharField(verbose_name=_('Name or Source Description'), max_length=225, blank=True)
+    url = models.ManyToManyField(ReferenceUrls, verbose_name=_('Add url'), blank=True)
+
+
+class InformalCountryDistrict(models.Model):
+    from api.models import Country, District
+
+    countries = models.ForeignKey(
+        Country, verbose_name=_('countries'), on_delete=models.CASCADE,
+        related_name='informal_country'
+    )
+    districts = models.ForeignKey(
+        District, verbose_name=_('districts'), on_delete=models.CASCADE,
+        related_name='informal_district'
+    )
+
+
+class ShareChoices(IntEnum):
+    IFRC = 1
+    RCRC = 2
+    RCRC_DONOR = 3
+
+    class Labels:
+        IFRC = _('IFRC Secretariat')
+        RCRC = _('RCRC Network')
+        IFRC_DONOR = _('RCRC Network and Donors')
+
+
+class InformalUpdate(models.Model):
+    '''
+    This is a base model for Informal Update
+
+    '''
+    from api.models import DisasterType
+
+    created_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL, verbose_name=_('created by'), related_name='informal_update_created_by',
+        null=True, blank=True, on_delete=models.SET_NULL,
+    )
+    modified_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL, verbose_name=_('modified by'), related_name='informal_update_modified_by',
+        null=True, blank=True, on_delete=models.SET_NULL,
+    )
+    created_at = models.DateTimeField(verbose_name=_('created at'), auto_now_add=True)
+    modified_at = models.DateTimeField(verbose_name=_('updated at'), auto_now=True)
+
+    # context
+    hazard_type = models.ForeignKey(
+        DisasterType, verbose_name=_('hazard type'), related_name='informal_update_hazard_type',
+        null=True, on_delete=models.SET_NULL
+    )
+    title = models.CharField(max_length=300)
+    situational_overview = models.TextField(verbose_name=_('Situational Overview'))
+
+    # Focal Point
+    originator_name = models.CharField(verbose_name=_('name'), max_length=100)
+    originator_title = models.CharField(verbose_name=_('title'), max_length=300)
+    originator_email = models.CharField(verbose_name=_('email'), max_length=300)
+    originator_phone = models.CharField(verbose_name=_('phone'), max_length=50, blank=True)
+
+    ifrc_name = models.CharField(verbose_name=_('name'), max_length=100)
+    ifrc_title = models.CharField(verbose_name=_('title'), max_length=300)
+    ifrc_email = models.CharField(verbose_name=_('email'), max_length=300)
+    ifrc_phone = models.CharField(verbose_name=_('phone'), max_length=50, blank=True)
+
+    # Share with
+    visibility = EnumIntegerField(ShareChoices, verbose_name=_('share with'), default=1)
```

Comment:
lets not provide default here!!

Reply:
For now let us keep default( recommended )